### ISSUE TITLE
Generated code for css-mask results in false negatives

### ISSUE THREAD

**Issue by lsirivong** [Modernizr/the-old-modernizr.com#34]
The css-mask feature adds the following line to the generated modernizr download:

```
Modernizr.addTest('cssmask', Modernizr.testAllProps('mask-repeat'));
```

This does not return 'true' as expected on Chrome 30.0.1599.101 or Safari 6.0.5

According to [Modernizr issue 747](https://github.com/Modernizr/Modernizr/issues/747) the testAllProps parameter should be camel cased to fix the issue.

```
Modernizr.addTest('cssmask', Modernizr.testAllProps('maskRepeat'));
```

**Comment by patrickkettner** [comment]
Hey @lsirivong, thanks a ton for the bug report, but this is the wrong place to open it at.
You can the [Modernizr](https://github.com/Modernizr/Modernizr) repo, this is the repo for the website. 

I'll look into this straight away, though.

**Comment by lsirivong** [comment]
Ah sorry Patrick, I thought since that this is an issue with the builder[http://modernizr.com/download/#-testallprops-css_mask] that it should go here.

**Comment by patrickkettner** [comment]
@lsirivong this was fixed in v2.6.3
